### ISSUE TITLE
LatentUpscale node needs BatchNorm

### ISSUE THREAD

**Issue by liuxiaolong98** [comfyanonymous/ComfyUI#4515]
### Feature Idea

The image, after being encoded by VAE, is transformed into a latent tensor that conforms to the standard Gaussian distribution. However, directly scaling the latent tensor does not follow this rule, which will lead to a lack of practicality in subsequent operations. We hope that the official can add the BatchNorm attribute to the LatentUpscale node

### Existing Solutions

```
norm_layer = nn.BatchNorm2d(feature_dim)
norm_upscale_latent = norm_layer(upscale_sample)
```
